it/uk/gov/hmrc/economiccrimelevyreturns/NotableErrorISpec.scala

```diff
@@ -43,4 +43,17 @@ class NotableErrorISpec extends ISpecBase with AuthorisedBehaviour {
     }
   }
 
+  s"GET ${routes.NotableErrorController.notRegistered().url}" should {
+
+    "respond with 200 status and the 'not registered' HTML view" in {
+      val result = callRoute(FakeRequest(routes.NotableErrorController.notRegistered()))
+      status(result) shouldBe OK
+
+      // See the "notRegistered.body" value as defined in the "messages.en" file
+      html(result) should include("You need to register for the Economic Crime Levy")
+
+      // See the "urls.registration" value as defined in the "application.conf" file
+      html(result) should include("/register-for-the-economic-crime-levy")
```

Comment:
I don't think we need the comments otherwise we would need to add comments on all our integration tests 🙂